roles/jd-server/src/main.rs

```diff
@@ -22,66 +27,147 @@ pub type Message = PoolMessages<'static>;
 pub type StdFrame = StandardSv2Frame<Message>;
 pub type EitherFrame = StandardEitherFrame<Message>;
 
-const BLOCK_REWARD: u64 = 5_000_000_000;
+const BLOCK_REWARD: u64 = 625_000_000;
```

Comment:
This should be automatically updated. Open an issue for it.

Reply:
After checking spec I realized that we already get the full amount from the TP. Why we need this const? Maybe should be removed.

Reply:
It's needed just to build the TxOut struct which is then modified with the `value_remaining` got from TP.
I can delete the constant and put a `0` value here if you want.

Reply:
Yep maybe is the best option. So we can check before sending the TxOut if the value is 0 and if it is it means that for whatever reason the value has not been changed so we fail. We should also add a comment where we build the TxOut that explain it.